db/migrate/20230927140030_add_posts_count_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostsCountToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :postsCounter, :integer
+  end
+end
```

Comment:
- [ ] Kindly organize your migrations files so that the project migrations can be run serially as they appear. This migration is the first migration. It is used to add a column to the users' table, but the users' table is created after this file, so when Rails comes across this file there is no users table yet, so it raises error "relation "users" does not exist". Kindly fix that. You may need to delete this migration and create it so that it uses the current date. There is already `posts_counter` in the users table so I don't think this migration is necessary.

  If you want to recreate this error then you need to drop your database and run the migrations again.